### ISSUE TITLE
API V1 tokens will be removed on August 31, 2023

### ISSUE THREAD

**Issue by arthurnw** [singer-io/tap-appsflyer#44]
Appsflyer will be removing API V1 tokens on August 31. This banner appears when logged in to Appsflyer:

> Caution!
>
> API token V1 will be removed on August 31, 2023. Please replace all instances of API token V1 with our new API token V2 and update the relevant APIs.
> 
> [Learn more](https://support.appsflyer.com/hc/en-us/articles/12399683708305-Bulletin-API-token-changes?utm_source=hq1&utm_medium=ui&utm_campaign=apiaccess)

The endpoint URLs are different:

- V1: `https://hq.appsflyer.com/export/app-id/installs_report/v5?`
- V2: `https://hq1.appsflyer.com/api/raw-data/export/app/app-id/installs_report/v5?` 

These probably need to change: https://github.com/singer-io/tap-appsflyer/blob/744a803519e1e2d5cf9ea78d9320850157f3f5ee/tap_appsflyer/__init__.py#L32-L36

The authentication method also needs to change (i.e. passing a token in the header instead of in the URL): https://github.com/singer-io/tap-appsflyer/blob/744a803519e1e2d5cf9ea78d9320850157f3f5ee/tap_appsflyer/__init__.py#L158

**Comment by cspaull-motorway** [comment]
Hi Was wondering if this issue was being looked at and whether the urls will be updated?

**Comment by christinesako** [comment]
Yes I am also wondering if this is planning to be fixed.

**Comment by robbiesch** [comment]
+1

**Comment by jailtojs** [comment]
Hello!
I use this API through Stitch Data.
There is a connection with this project.
I've been having problems with this issue

**Comment by dennysrega1ado** [comment]
This tap has stopped working since September 18. 
Is there any plan to upgrade from V1 to V2?

```bash
401 - b"API token V1 was removed on September 18, 2023 in favor of the more secure API token V2.Please replace all instances of API token V1 with our new API token V2 and update the relevant APIs.<a href='https://support.appsflyer.com/hc/en-us/articles/12399683708305-Bulletin-API-token-changes?query=token v1'>Learn more</a>"]
```

**Comment by maupangea** [comment]
+1

**Comment by syed-a-ali** [comment]
+1

**Comment by KeltonKarboviak** [comment]
We are also currently affected by this and not able to ingest data via Stitch since September 18th

**Comment by rdeshmukh15** [comment]
yes, looking into this issue

**Comment by rdeshmukh15** [comment]
Released the changes as part of v0.1.0
